### PR TITLE
balance: adjust balance config.

### PR DESCRIPTION
**Pull request by disksing**
Now tikv-server sends a region heartbeat to PD as soon as pending peer catches up with leader, the old limit values turn out to be too aggressive.

### CODE REVIEW

File: conf/config.toml [hunk 1]
```diff
@@ -57,10 +57,10 @@ max-merge-region-size = 0
 max-snapshot-count = 3
 max-pending-peer-count = 16
 max-store-down-time = "1h"
```
Comment (nolouch) [inline conf/config.toml]: my mistake, and here should be 30m.

**Comment by Connor1996** [comment]
please fix losing max-pending-peer-count in clone()

File: conf/config.toml (hunk 1 above)
Comment (disksing) [reply conf/config.toml]: ok.

**Comment by disksing** [comment]
tikv-server panics, restarting the test.

**Pull request merged**